lib/PuppeteerSharp/Downloader.cs

```diff
@@ -21,6 +21,8 @@ public class Downloader
         };
         private string _downloadHost;
 
+        public const int DefaultRevision = 526987;
```

Comment:
I can buy this now. But I created an issue to make it configurable in v1.0 https://github.com/kblok/puppeteer-sharp/issues/84